pyproject.toml

```diff
@@ -49,6 +50,7 @@ docs = [
     "trimesh <4.6",
     "gltflib",
     "imgui-bundle>=1.6.0",
+    "httpx",
```

Comment:
do we have examples that are built by docs that require httpx? I'm not sure if that's desirable.

Reply:
The "iridescence.py" example utilizes httpx (via the glTF loader) to load sample models from remote sources. Future examples may also load resources remotely, eliminating the need to include these assets in the pygfx repository. Of course, we can disable automatic testing and documentation generation for this type of example, but i think it's worthwhile to include certain examples and demos that showcase distinctive features in the documentation. :)